test/PushNotificationTests/APITests.cpp

```diff
@@ -262,6 +262,11 @@ namespace Test::PushNotifications
             RunTest(L"ActivatorTest", testWaitTime());
         }
 
+        TEST_METHOD(ActivatorTest_Unpackaged)
```

Comment:
Should this be RegisterActivatorTest_Unpackaged?

Reply:
ActivatorTest tests both Register/Unregister.